docs/conditions.rst

```diff
@@ -36,3 +253,63 @@ Conditions support the following operators:
 
 
 ..
+
+
+FAQ
+***
+
+| **Q:** In a multiple condition scenario when 1 event matches and the rest of the events don't match
+| how long does the Rule engine keep the previous event around?
+
+| **Ans:** Currently there is no time limit on how long the rule engine keeps the matched event.
+| Once they match they are retracted.
+
+| **Q:** When does the Ansible rulebook stop processing?
+
+| **Ans:** When a Shutdown event is generated from the source plugin, or shutdown action is invoked.
+
+| **Q:** Will a condition be evaluated if a variable is missing?
+
+| **Ans:** If a condition refers to an object.attribute which doesn't exist then that condition
+| is skipped and not processed.
+
+Example::
+
+   name: send to debug
+   condition: event.payload.eventType != 'GET'
+   action:
+        debug:
+
+
+   In the above case if any of the event.payload.eventType is undefined the condition is
+   ignored and doesn't match anything.
+
+| **Q:** When a rule book has multiple rule sets and one of them shuts down are all rule sets terminated?
+
+| **Ans:** Yes, so care should be taken if there are any playbooks running in the other rule sets
+
+| **Q:** How do I check if an attribute in an object referred in a condition exists?
+
+| **Ans:** Use the is defined
+
+Example::
+
+      name: rule1
+      condition: event.msg is defined
+      action:
+        retract_fact:
+          fact:
+            msg: "{{event.msg}}"
+
+| **Q:** How do I check if an attribute in an object referred in a condition does not exist?
+
+| **Ans:** Use the is not defined
```

Comment:
```suggestion
| **Ans:** Use the "is not defined" logical operator
```

Reply:
To be strict, it's an identity operator. Logical operators would "and" and "or"

Reply:
Fixed